src/test/TestHelper.kt

```diff
@@ -118,15 +118,17 @@ object TestHelper {
         }
     }
 
-    fun generateUserAccounts(count: Int = 1): List<UserAccount> {
-        var userAccounts = mutableListOf<UserAccount>()
+    fun generateUserAccounts(count: Int = 1): Map<String, UserAccount> {
```

Comment:
maybe this should return a NewUserAccount obj instead of this map -- so that way the tests will have access to the priv key and secret

Reply:
List<NewUserAccount>